src/components/SudokuGrid/SudokuGrid.js

```diff
@@ -7,12 +7,12 @@ export default function SudokuGrid({
 }) {
   return (
     <div className="Sudoku__grid">
```

Comment:
im not very sure about that classname... is grid a block itself?  i mean, you call your component "SudokuGrid"... so i would go for the same classname.

Reply:
Thanks for pointing out!
I think by convention the class name of the root element of a component should have the same name as the component.
I will change it.